docs/partner_management_service_api_documentation.json

```
@@ -10158,13 +10149,13 @@
                 }
             }
         },
-        "/partners/{partnerId}/policies/{policyId}/credential-types-request": {
+        "/partner-policy-requests/{requestId}/credential-types-request": {
```

Biome (2.4.15)

[error] The key /partner-policy-requests/{requestId}/credential-types-request was already declared.

(lint/suspicious/noDuplicateObjectKeys)